src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise(function (resolve, reject) {
+  // eslint-disable-next-line no-shadow
+  document.getElementById('logo').addEventListener('click', function (event) {
```

Comment:
Select logo element outside the promise1 scope, just create a constant for it.

Reply:
I don't understand what exaclly i need to do :(